.github/workflows/ci.yml

```diff
@@ -28,3 +28,12 @@ jobs:
        run: nix-build --no-out-link ./nix -A devEnv
      - name: Install the wire-server-direnv
        run: nix-env -if ./nix -A devEnv
+     - name: Build docs
+       run: nix-build --no-out-link ./nix -A docs
+     - name: Deploy docs
+       if: github.ref == 'refs/heads/master'
+       run: |
+         cd docs
+         docs=$(nix-build --no-out-link ./nix -A docs)
+         aws s3 sync $docs/html s3://origin-docs.wire.com/
+         aws s3 cp $docs/pdf/wire_federation.pdf s3://origin-docs.wire.com/main.pdf
```

Comment:
We probably still needs secrets to successfully deploy, but let's do this once we agree on the rest of the PR.